src/Post.php

```diff
@@ -350,6 +350,7 @@ public function get_og_image() {
 		return [];
 	}
 
+
```

Comment:
Not a big deal, but this extra white space is not needed 😉

Reply:
😄 Done.